src/utils/project/archiveProject.js

```diff
@@ -0,0 +1,10 @@
+import { removeDir } from '@tauri-apps/api/fs';
+import getProjectPath from '../paths/getProjectPath';
+
+export default async function archiveProject(id) {
```

Comment:
Should maybe rename this to just `removeProject`? To avoid confusion with archiving the action.